moc_openstack_api_reporting.py

```diff
@@ -727,17 +727,19 @@ def main():
 
     vm_keys = list(cluster_state["vm_timestamps"].keys())
     for key in vm_keys:
-        if cluster_state["vm_timestamps"][key]["updated"] == 0:
-            del cluster_state["vm_timestamps"][key]
-        else:
-            del cluster_state["vm_timestamps"][key]["updated"]
+        if "updated" in cluster_state["vm_timestamps"][key]:
+            if cluster_state["vm_timestamps"][key]["updated"] == 0:
+                del cluster_state["vm_timestamps"][key]
+            else:
+                del cluster_state["vm_timestamps"][key]["updated"]
 
     vol_keys = list(cluster_state["vol_timestamps"].keys())
     for key in vol_keys:
-        if cluster_state["vol_timestamps"][key].get("updated") == 0:
-            del cluster_state["vol_timestamps"][key]
-        else:
-            del cluster_state["vol_timestamps"][key]["updated"]
+        if "update" in cluster_state["vol_timestamps"][key]:
```

Comment:
I believe this is a typo here as it should be `"updated"`

Reply:
yes it is a typo.  fixed.